src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,109 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double RESIZE = 1.5;
+    private Object[] elementsData;
+    private int size = 0;
+
```

Comment:
assigining to 0 is redundant for the class fields, this is default value for int type variables